go.sum

```diff
@@ -61,8 +61,8 @@ github.com/crossplaneio/crossplane v0.5.0-rc.0.20191211203849-05517d46525d h1:Xi
 github.com/crossplaneio/crossplane v0.5.0-rc.0.20191211203849-05517d46525d/go.mod h1:f/e99LDWClmhkvMuvK7UMP1g5b718DGmdnpdrflK0HI=
 github.com/crossplaneio/crossplane-runtime v0.2.4-0.20191211004842-fa83d075257c h1:Xmv6RSdmFf2WeBpzVaqEBVxmGVAAUxA61eMgrdQ+Vsw=
```

Comment:
similar to https://github.com/crossplaneio/stack-azure/pull/85/files#r362299283:
there are still refs to crossplane-runtime v0.2.4. Should we do something similar to what we did in https://github.com/crossplaneio/stack-rook/pull/32/files#r361492505, or just treat this as a simple backport only that does not introduce additional changes to the release branch outside of the commit being backported?